src/tests/ftest/util/fio_test_base.py

```diff
@@ -87,16 +89,27 @@ def _create_cont(self):
         # TO-DO: Enable container using TestContainer object,
         # once DAOS-3355 is resolved.
         # Get Container params
+        # Not enabling Test Container for FIO due to
+        # intermittant failures daos_dfuse creation. (DAOS-4172)
         # self.container = TestContainer(self.pool)
         # self.container.get_params(self)
 
         # create container
         # self.container.create()
         env = Dfuse(self.hostlist_clients, self.tmp).get_default_env()
         # command to create container of posix type
-        cmd = env + "daos cont create --pool={} --svc={} --type=POSIX".format(
-            self.pool.uuid, ":".join(
-                [str(item) for item in self.pool.svc_ranks]))
+        if self.co_prop is None:
+            cmd = env + ("daos cont create --pool={} --svc={} "
+                         "--type=POSIX".format(
+                            self.pool.uuid, ":".join(
+                                [str(item) for item in self.pool.svc_ranks])))
+        else:
+            cmd = env + ("daos cont create --pool={} --svc={} --type={} "
+                         "--properties=cksum:{},cksum_size:{}".format(
+                            self.mdtest_cmd.dfs_pool_uuid.value,
+                            self.mdtest_cmd.dfs_svcl.value,
```

Comment:
(pylint-bad-continuation) Wrong hanging indentation (add 1 space).
(pylint-no-member) Instance of 'FioBase' has no 'mdtest_cmd' member